src/components/drop-down/drop-down.scss

```diff
@@ -0,0 +1,69 @@
+@import '../../assets/colors';
+@import '../../assets/styles';
+
+.drop-down {
+  @include flex-center;
+
+  > label {
+    margin-right: 8px;
+
+    &.light {
+      color: $blue;
+    }
+
+    &.dark {
+      color: $white;
+    }
+  }
+}
+
+// avoid nesting in .drop-down to make menu more easily targetable
+.drop-down-menu {
+  position: relative;
+  width: 150px;
+
+  > button {
+    width: 100%;
+    padding: 5px 0px 5px 15px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .options {
+    width: 100%;
+    margin-top: 2px;
+    position: absolute;
+    z-index: 1;
+    background-color: $white;
+    border-radius: $standard-radius + 1px; //hides corners of div
+    @include box-shadow;
+
+    .option {
+      width: 100%;
+      box-sizing: border-box;
+      border-left: 1px solid $flashcard-white;
+      border-right: 1px solid $flashcard-white;
+      background-color: $white;
+      overflow: scroll;
```

Comment:
This might be problematic, this would make the scrollbars always show -- we also probably don't want text to go past the option or label.

Here's what it looks like now:
![current-overflow](https://user-images.githubusercontent.com/29434693/160180587-73f7d0bc-bce0-4a1c-9b1c-9a155f093c1c.png)

We could try something with overflow ellipsis (perhaps make a mixin for this) and apply this to this `.option` here as well as `.drop-down-menu > button > label`
```css
overflow: hidden;
white-space: nowrap;
text-overflow: ellipsis;
```

![fixed-overflow](https://user-images.githubusercontent.com/29434693/160180771-ddccdcce-db7f-45dd-b0f7-590f9dec3833.png)

Reply:
Another related note, it doesn't seem like the drop-down-menu label is picking up the font? Perhaps we need to add the font-family to it again 😕

Reply:
haha, yeah that looks awful, I got that fixed to use the ellipsis styling now. Weird that it wasn't showing up on my browser.